src/reducers/search.js

```diff
@@ -38,6 +38,7 @@ export const initialState = {
     displayName: 'Top inspiration',
     id: 'marketing:homepage.dk.spies'
   },
+  scrollPage: 6,
```

Comment:
In fact, I'd be tempted to maybe start it at zero (since we do start with zero results), and then update it at L87 where we fetch the 30 items. Then that way at least the 30 and the 6 are close.

Or possibly even better we could set it to `Math.ceil(itemsToDisplay.length / 5)` so that the provenance is clear.